### PR TITLE
FB-321-trial-version

### PR DESCRIPTION
**Pull request by paulinabjacykowska**
Changes are described in the https://vazco1.atlassian.net/browse/FB-321

### CODE REVIEW

File: src/sections/TestProductSection/TestProductSection.tsx [hunk 1]
```diff
@@ -13,16 +14,27 @@ export const TestProductSection = () => {
...
         </p>
-        <Link to="/demo" internal>
-          <Button>Check how it works</Button>
+        <List>
+          <ListItem>how easy is it to integrate Forminer</ListItem>
+          <ListItem>the quality of our code.</ListItem>
+        </List>
+        <p>
+          * If you want to use Forminer for a non-production environment (no
+          users on the platform), you can use it to your heart's content.
```
Comment (mkaliszewski) [inline src/sections/TestProductSection/TestProductSection.tsx:34]: ESLint: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.(react/no-unescaped-entities)

```suggestion
          users on the platform), you can use it to your heart&apos;s content.
```

File: src/sections/TestProductSection/TestProductSection.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { Button } from '../../components/Button';
 import { Link } from '../../components/Link';
 import { SectionLayout } from '../../components/SectionLayout';
 import { TextAndMediaItem } from '../../components/TextAndMediaItem';
+import { List, ListItem } from '../../components/List';
```
Comment (mkaliszewski) [inline src/sections/TestProductSection/TestProductSection.tsx:8]: ESLint: `../../components/List` import should occur before import of `../../components/SectionLayout`

```suggestion
import { List, ListItem } from '../../components/List';
import { SectionLayout } from '../../components/SectionLayout';
import { TextAndMediaItem } from '../../components/TextAndMediaItem';
```

File: src/sections/TestProductSection/TestProductSection.tsx [hunk 3]
```diff
@@ -13,16 +14,27 @@ export const TestProductSection = () => {
...
+          During the trial, you can assess the following:
         </p>
-        <Link to="/demo" internal>
-          <Button>Check how it works</Button>
+        <List>
+          <ListItem>how easy is it to integrate Forminer</ListItem>
+          <ListItem>the quality of our code.</ListItem>
+        </List>
+        <p>
+          * If you want to use Forminer for a non-production environment (no
```
Comment (mkaliszewski) [inline src/sections/TestProductSection/TestProductSection.tsx:33]:
```suggestion
          *If you want to use Forminer for a non-production environment (no
```

No space after `*` on the mockup

**Review by mkaliszewski — CHANGES_REQUESTED** [review]
Only minor stuff.

We also have `CustomBulletList` & `CustomBulletListItem`, are you sure we shouldn't use it here?

Screenshot from 2023-07-03 13-10-48

File: src/sections/TestProductSection/TestProductSection.tsx (hunk 1 above)
Comment (paulinabjacykowska) [reply src/sections/TestProductSection/TestProductSection.tsx:34]: Haha, I thought for a really long time if I should use any symbols entities here but I didn't get any warnings so I decided not to 🙈 But I am changing it now!

**Comment by paulinabjacykowska** [comment]
> Only minor stuff.
> 
> We also have `CustomBulletList` & `CustomBulletListItem`, are you sure we shouldn't use it here?
> 
> Screenshot from 2023-07-03 13-10-48

I saw it but Krzysiek said that we want to use plain black bullets for now

**Review by mkaliszewski — APPROVED** [review]
Nice, thanks for the changes

**Pull request merged**